velox/functions/sparksql/In.cpp

```diff
@@ -130,6 +130,22 @@ void registerInFn(const std::string& prefix) {
       {prefix + "in"});
 }
 
+void registerInFnForShortDecimal(const std::string& prefix) {
+  registerFunction<
+      InFunctionOuter<ShortDecimal<P1, S1>>::template Inner,
+      bool,
+      ShortDecimal<P1, S1>,
+      Array<ShortDecimal<P1, S1>>>({prefix + "in"});
+}
+
+void registerInFnForLongDecimal(const std::string& prefix) {
```

Comment:
To reduce duplicate, can we combine them into 'registerInFnForDecimal' and register for short and long decimal types in the 'registerIn'?